### PR TITLE
Add NodeSource note for Ubuntu and Debian based systems

### PR DESCRIPTION
**Pull request by BEERproductions**
Many People use Ubuntu and Debian Server distributions, so something like this would be really good to include in the setup.

### CODE REVIEW

**Review by bitfl0wer — CHANGES_REQUESTED** [review]
This addition is misplaced. The text you added should, in my opinion, be placed at the "Node JS" bulletpoint, not the "On Linux" bulletpoint.

File: docs/setup/server/index.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ We do not recommend using Windows to run {{ project.name }}.
 -   [Python](https://www.python.org/). Version 3.10 or later. Make sure this is executable via `python` in your terminal.  
      (See: `python-is-python3` package)
 -   On Linux: `gcc`/`g++`. Packaged with `build-essential` on Debian/Ubuntu and `base-devel` on Arch.
+     (NOTE: Ubuntu or Debian based systems often ship with outdated versions, so we recommend using NodeJS from [NodeSource](https://github.com/nodesource/distributions))
```
Comment (Puyodead1) [inline docs/setup/server/index.md:21]:
```suggestion
     (NOTE: Ubuntu/Debian based systems often ship with outdated node versions, you can use [NodeSource](https://github.com/nodesource/distributions) to install updated an updated version.)
```

**Review by Puyodead1 — CHANGES_REQUESTED** [review]

**Comment by Puyodead1** [comment]
> This addition is misplaced. The text you added should, in my opinion, be placed at the "Node JS" bulletpoint, not the "On Linux" bulletpoint.

I agree

**Comment by BEERproductions** [comment]
Well both options in my opinion is fine, but if you both think that it belongs under NodeJS, sure

File: docs/setup/server/index.md (hunk 1 above)
Comment (Puyodead1) [inline docs/setup/server/index.md:21]:
```suggestion
     (NOTE: Ubuntu/Debian based systems often ship with an outdated version of node, you can use [NodeSource](https://github.com/nodesource/distributions) to install a newer version.)
```

**Review by Puyodead1 — CHANGES_REQUESTED** [review]

File: docs/setup/server/index.md (hunk 1 above)
Comment (Puyodead1) [reply docs/setup/server/index.md:21]: I think this wording sounds better

Comment (BEERproductions) [reply docs/setup/server/index.md]: I agree

Comment (Puyodead1) [reply docs/setup/server/index.md]: > I agree

could we change `Ubuntu or Debian` to at least `Ubuntu and Debian`?

**Pull request merged**